docs/source/support/faq.rst

```diff
@@ -0,0 +1,183 @@
+==========================
+Frequently Asked Questions
+==========================
+
+What is the recommended specifications for running a nucypher?
+-------------------------------------------------------------------
+
+Worker nodes need to run ``nucypher`` and a local ethereum node, currently on the Goerli network. In total, you will
+require at 4GB for RAM. Nodes also need 24/7 uptime and a static, public IPv4 address.
+
+For ``nucypher`` specific requirements, see `System Requirements and Dependencies <https://docs.nucypher.com/en/latest/guides/installation_guide.html#system-requirements-and-dependencies/>`_.
+
+
+What is the network name for Incentivized Testnet?
+--------------------------------------------------
+
+The network name is ``cassandra``.
+
+
+How long is a period?
+---------------------
+
+1 period equals 24 hours. Periods begin at midnight UTC time.
+
+
+What's the difference between staker address and worker address? Can they be the same?
+--------------------------------------------------------------------------------------
+
+Technically, yes, but it is not recommended. The accounts have different security considerations - the staker address can be a high-value hardware wallet
+(staker with NU) that performs stake management while the worker is low-value and needs to remain
+unlocked while running (software wallet) since it used by an Ursula node.
+
+You should stake with one address and set the worker to be a different address Subsequently, you can bond
```

Comment:
```suggestion
You should stake with one address and set the worker to be a different address. Subsequently, you can bond
```
Also, perhaps mention that both need ETH and only the staker needs NU.

Reply:
✔️